.github/resources/apiserver/root-ca-configmap.yaml

```diff
@@ -0,0 +1,40 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: root-ca
+  managedFields:
+    - manager: Mozilla
+      operation: Update
+      apiVersion: v1
```

Comment:
can you remove all occurrences of `managedFields` in any of the manifests, this is not required